### ISSUE TITLE
Training with transformer models

### ISSUE THREAD

**Issue by NikhilCherian** [rgcottrell/pytorch-human-performance-gec#10]
@rgcottrell @tianfeichen @cqlijingwei 

I am actually researching about the different types of models for grammatical correction.

So, i used arch='transformer_iwslt_de_en' from this link, 
https://github.com/pytorch/fairseq/tree/master/examples/translation#iwslt14-german-to-english-transformer

Here, i changed the model from fconv to transformer. I got really disappointing results for the test set, jfleg.

![image](https://user-images.githubusercontent.com/11813426/83130896-50162080-a0df-11ea-9f9f-a980a52f5d7b.png)

Do you want to share some thoughts? What newer features from fairseq do you think will be the fittest for the extension of your project? 
Any help would be appreciated. Thanks in advance.
